### PR TITLE
Add FP&A role

### PR DESCRIPTION
**Pull request by dadlerj**
Draft until the job link has been created in Lever

### CODE REVIEW

File: job-descriptions/financial-planning-analysis.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### In the first 90 days you will...
+
+- Use existing materials to build a quarterly 1-year and 3-year forward-looking models to be shared with our Board of Directors on an ongoing basis
+  - Work with our go-to-market operations and business operations team.
+- Take ownership of our quarterly (and annual) financial planning and help us establish an internal budgeting process.
+- Be deeply involved if/when any strategic finance discussions arise, including fundraising, M&A, debt financing, and more.
+- Take on at least one strategic initiative to improve our financial systems and processes.
+
+## Qualifications
+
```
Comment (ebrodymoore) [inline job-descriptions/financial-planning-analysis.md:30]:
```suggestion
- An excitement for software development (no training as an Engineer required, but you should have an understanding of how software development happens and Sourcegraph's value proposition).
- Demonstrated background in strategic or operational finance functions roles (Finance, BizOps, or Accounting at a technology company, Consulting, Investment Banking, etc.)
```

**Comment by ebrodymoore** [comment]
@dadlerj why not just Financial Analyst?

**Comment by dadlerj** [comment]
FP&A is actually a specific role/function — not just a generic term that I came up with for this role!

**Comment by ebrodymoore** [comment]
Sounds good, I just wasn't sure if you wanted to frame it as broader than FP&A!

**Pull request merged**